Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -888,7 +887,7 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
         
         do
         {
-            try imageData.write(to: URL(fileURLWithPath: path), options: .atomic)
+            try imageData?.write(to: URL(fileURLWithPath: path), options: .atomic)
```

Comment:
Optional chaining can cover up the exception if the image type is not supported?

Reply:
updated